gradle-retrolambda/src/main/groovy/me/tatarka/RetrolambdaPluginAndroid.groovy

```diff
@@ -103,27 +104,27 @@ class RetrolambdaPluginAndroid implements Plugin<Project> {
     }
 
     private static configureCompileJavaTask(Project project, BaseVariant variant, RetrolambdaTransform transform) {
-        variant.javaCompile.doFirst {
+        variant.javaCompileProvider.get().doFirst {
```

Comment:
By using `get()` we break the lazy configuration the provider introduces, should figure out another way to do it, probably with `configure`? https://docs.gradle.org/current/userguide/task_configuration_avoidance.html